website/docs/exempt-namespaces.md

```diff
@@ -65,6 +67,10 @@ If it becomes necessary to exempt a namespace from Gatekeeper webhook entirely (
    3. Add the `admission.gatekeeper.sh/ignore` label to the namespace. The value attached
       to the label is ignored, so it can be used to annotate the reason for the exemption.
 
+Similarly, you can also exempt entire groups of namespaces using the `--exempt-namespace-prefix` and `--exempt-namespace-suffix` flags.
+
+As an example, if you wanted to exempt all namespaces ending with `system` such as `kube-system`, `gatekeeper-system`, etc. you could set the flag `--exempt-namespace-suffix=-system`
```

Comment:
This would not directly exempt the namespace, the user would need to add the `admission.gatekeeper.sh/ignore` label to the namespace as well (setting this flag is what lets that label be set).

Also, this will only exempt namespaces from the webhook.

The purpose of the flag is to be able to bypass the webhook entirely, so there is no chance G8r interferes with the operation of the namespace. This is b/c there is a label selector in the ValidatingAdmissionWebhookConfiguration.

Reply:
https://github.com/open-policy-agent/gatekeeper/blob/1076798df31db7c58cfd9def5f4972325fb519b8/pkg/webhook/namespacelabel.go#L27-L29